pytorch_lightning/plugins/precision/apex_amp.py

```diff
@@ -47,13 +48,17 @@ def main_params(self, optimizer: Optimizer) -> _PARAMETERS:
 
     def dispatch(self, trainer: "pl.Trainer") -> None:
         if not self._connected:
-            accelerator = trainer.accelerator
-            _, accelerator.optimizers = amp.initialize(
-                trainer.lightning_module, accelerator.optimizers, opt_level=self.amp_level
+            strategy = trainer.training_type_plugin
+            _, strategy.optimizers = amp.initialize(
+                trainer.lightning_module, strategy.optimizers, opt_level=self.amp_level
             )
             self._connected = True
         return super().dispatch(trainer)
 
+    @property
+    def amp_backend(self) -> Optional[LightningEnum]:
+        return AMPType.APEX
```

Comment:
note, there is already a backend class attribute, so we should probably use that for now?